CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 About changelog [here](https://keepachangelog.com/en/1.0.0/)
 
+## [Unreleased]
```

Comment:
The changelog item of this PR should be consistent with the other ones. Please add 
`- OMIM table is scrollable if higher than 700px` to the list of items down, from line 14 of this file

Reply:
I have updated the changelog,I hope I understood the request correctly